### ISSUE TITLE
yml parse error?

### ISSUE THREAD

**Issue by atsushi-ishibashi** [k1LoW/runn#299]
when you define vars with double quote, it seems to ignore yml config? The result is the same when single quote.
```
desc: basicauth
vars:
  user: "hoge"
  password: "pass@?$"
steps:
  basicAuth:
    bind:
      basicToken: base64encode(vars.user + ':' + vars.password)
  output:
    dump: basicToken
```
```
$ runn run ./book.yml --debug
Generated by `runn new` ... ok

1 scenario, 0 skipped, 0 failures
```

if you remove double quote, it works well
```
desc: basicauth
vars:
  user: hoge
  password: pass@?$
steps:
  basicAuth:
    bind:
      basicToken: base64encode(vars.user + ':' + vars.password)
  output:
    dump: basicToken
```
```
$ runn run ./book.yml --debug
Run 'bind' on 'basicauth'.steps.basicAuth

Run 'dump' on 'basicauth'.steps.output
aG9nZTpwYXNzQD8k
basicauth ... ok

1 scenario, 0 skipped, 0 failures
```

**Comment by k1LoW** [comment]
Seems to be an issue with https://github.com/k1LoW/expand

https://github.com/k1LoW/runn/blob/51e59249aa179bc0668279a319f6e7aa8f0ce6d4/runbook.go#L66

I'll check it out.

**Comment by k1LoW** [comment]
Looks like k1loW/expand is responding to `$`.

**Comment by atsushi-ishibashi** [comment]
you're right. the below works well
```
desc: basicauth
vars:
  user: "hoge"
  password: "pass@?"
steps:
  basicAuth:
    bind:
      basicToken: base64encode(vars.user + ':' + vars.password)
  output:
    dump: basicToken
```

**Comment by k1LoW** [comment]
Fixed in v0.49.3.
